lib/player.rb

```diff
@@ -0,0 +1,69 @@
+require_relative './scoring'
+require_relative './tilebag'
+
+module Scrabble
+    class Player < Scoring
```

Comment:
Why does `Player` inherit from `Scoring`? I don't think that the "is-a" relationship holds here - I don't think of a `Player` as being a type of `Scoring` (in fact that doesn't really make sense).

I think that composition is probably the relationship you want here - `Player` _calls_ `Scoring.score` as an external method when it needs something scored.